Rubberduck.Main/Extension.cs

```diff
@@ -300,5 +311,15 @@ private void ShutdownAddIn()
                 }
             }
         }
+
+        [DllImport("SHCore.dll", SetLastError = true)]
```

Comment:
Does this mean we won't be able to support Windows 7?

Reply:
I'd also be happier if this were declared elsewhere given that we're planning on rearranging these. Even if it's in one of the other "dll dumps".

Reply:
Apparantly all that's needed in Windows < 8.1 is to set the global DPI scaling to XP-style (in advanced settings).